src/scraper.rs

```diff
@@ -91,15 +91,47 @@ impl Scraper {
         old_url_str.push_str(&new_url_str);
     }
 
+    /// Find the charset of the webpage. ``data`` is not a String as this might not be utf8
+    fn find_charset(data: &[u8]) -> Option<String> {
+        let data_utf8 = unsafe { String::from_utf8_unchecked(Vec::from(data)) };
+        let regex = Regex::new("<meta.*charset\\s*=\\s*([^\"\\s]+).*>").unwrap();
```

Comment:
I think we can have this statically compiled, so we don't recreate the regex unnecessarily each time we call the function

Reply:
Also I'm gonna trust you on the regex lmao

Reply:
> I think we can have this statically compiled, so we don't recreate the regex unnecessarily each time we call the function

``regex!``has been removed. The recommended way is lazy_static. Fixed in fb3eac8